### ISSUE TITLE
Uncaught PHP Exception GuzzleHttp\Exception\RequestException: "cURL error 3: <url> malformed

### ISSUE THREAD

**Issue by pagestyles** [codefog/contao-instagram#22]
Hello, I am using the extension for the first time and get an error message:

It runs fine for a while, then after a few minutes the error chops the whole page

Contao 4.9.3
GuzzleHttp/6.5.5
curl/7.58.0
PHP/7.4.5

```
[2020-06-19 09:22:19] request.INFO: Matched route "tl_page.2.root". {"route":"tl_page.2.root","route_parameters":{"_token_check":true,"_controller":"Contao\\FrontendIndex::renderPage","_scope":"frontend","_locale":"de","pageModel":"[object] (Contao\\PageModel: {})","_route":"tl_page.2.root"},"request_uri":"https://www.z-connection.de/","method":"GET"} []
[2020-06-19 09:22:19] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2020-06-19 09:22:19] request.CRITICAL: Uncaught PHP Exception GuzzleHttp\Exception\RequestException: "cURL error 3: <url> malformed (see https://curl.haxx.se/libcurl/c/libcurl-errors.html)" at /www/htdocs/[REMOVED]/vendor/guzzlehttp/guzzle/src/Handler/CurlFactory.php line 201 {"exception":"[object] (GuzzleHttp\\Exception\\RequestException(code: 0): cURL error 3: <url> malformed (see https://curl.haxx.se/libcurl/c/libcurl-errors.html) at /www/htdocs/[REMOVED]/vendor/guzzlehttp/guzzle/src/Handler/CurlFactory.php:201)"} []
```

Stacktrace:
[contao-instagram_error.txt](https://github.com/codefog/contao-instagram/files/4803137/contao-instagram_error.txt)

**Comment by qzminski** [comment]
That is strange. Would you be able to provide me the FTP access details (e.g. via e-mail) in order for me to debug the problem directly on your website?

**Comment by pagestyles** [comment]
Hi, I e-mailed you the dates.
Additionally I have installed a tracker, with the following results. 
The extension runs stable since 4 days, but before that you can see the crashes. The logfiles are on the server.

Keyword found | 26/06/2020 15:26 | for 4 days
Keyword not found | 26/06/2020 13:30 | 2 hours
Keyword found | 26/06/2020 11:20 | 2 hours
Keyword not found | 26/06/2020 10:19 | an hour

**Comment by qzminski** [comment]
Sorry, I didn't get any e-mail from you. Can you re-send it please?

Also, what is this tracker and what do those messages mean?

**Comment by pagestyles** [comment]
Hi, I resent them again.
The tracker (trakked.io) just looks on the page to see if a certain element is present, if not it will be reported. at the end of the line you can see the second one how long the error existed or how long everything went well

**Comment by pagestyles** [comment]
did you get the email ?

**Comment by qzminski** [comment]
Sorry, I haven't received it. Did you sent it to kamil.kuzminski@codefog.pl or info@codefog.pl?

**Comment by pagestyles** [comment]
to both :-)
kamil.kuzminski@codefog.pl and info@codefog.pl

title is: guthub issue #22 codefog / contao-instagram

did you check the spam folder?

**Comment by qzminski** [comment]
I have found it in spam folder :+1: I will keep you updated about the progress on this one.

**Comment by qzminski** [comment]
I have just released 2.1.5 that should allow you to bypass this problem. You can find more information about it in the documentation: https://github.com/codefog/contao-instagram/blob/hotfix/2.1.5/docs/README.md#curl-error-3--malformed
